src/content/docs/apm/agents/php-agent/frameworks-libraries/guzzle.mdx

```diff
@@ -10,7 +10,7 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-New Relic supports versions 3, 4, 5, 6 and 7 of the <DNT>**Guzzle HTTP client library**</DNT> with [New Relic PHP agent version 7.0](/docs/release-notes/agent-release-notes/php-release-notes) or higher.
+New Relic supports versions 5, 6 and 7 of the <DNT>**Guzzle HTTP client library**</DNT> with [New Relic PHP agent version 7.0](/docs/release-notes/agent-release-notes/php-release-notes) or higher.
```

Comment:
```suggestion
New Relic supports versions 5, 6 and 7 of the <DNT>**Guzzle HTTP client library**</DNT>.
```
I think we can probably remove the mention of agent version 7.0 and the link to the general release notes page. I could see this possibly being misconstrued as supporting agent version 7.0